app/models/asset.rb

```diff
@@ -135,7 +135,7 @@ def reset_state
   end
 
   def schedule_virus_scan
-    self.delay.scan_for_viruses if self.unscanned?
+    VirusScanWorker.perform_async(self.id) if self.unscanned?
```

Comment:
Minor: I know you mentioned it in a previous commit note, but I think it would be nice to mention in the note for this commit that you're leaving `scan_for_viruses` executing the virus scanning synchronously to support any remaining Delayed Jobs in the queue.

Reply:
Thanks! I'll add a note to the commit message.